### PR TITLE
Added Authcode to panel_domains

### PR DESCRIPTION
**Pull request by megaspatz**
- added authcode to table panel_domains
- added authcode to domain-add and -edit page for admins
- added authcode to domain-edit page for customers

### CODE REVIEW

File: lib/formfields/customer/domains/formfield.domains_edit.php [hunk 1]
```diff
@@ -101,7 +101,12 @@
 						'label' => $lng['domain']['openbasedirpath'],
 						'type' => 'select',
 						'select_var' => $openbasedir
-					)
+					),
+                                        'authcode' => array(
```
Comment (d00p) [inline lib/formfields/customer/domains/formfield.domains_edit.php]: for the customer, i would hide this, if there is no authcode, e.g.
`visible => (!empty($result['authcode']))`

File: install/froxlor.sql [hunk 2]
```diff
@@ -246,6 +246,7 @@ CREATE TABLE `panel_domains` (
   `add_date` int( 11 ) NOT NULL default '0',
   `registration_date` date NOT NULL,
   `termination_date` date NOT NULL,
+  `authcode` varchar(255) NOT NULL DEFAULT '',
```
Comment (d00p) [inline install/froxlor.sql]: this file also holds the current version number which needs to be adjusted (see values for panel_settings)

Comment (megaspatz) [inline install/froxlor.sql]: Ups, i forgot this. -> Done.

File: lib/formfields/customer/domains/formfield.domains_edit.php (hunk 1 above)
Comment (megaspatz) [inline lib/formfields/customer/domains/formfield.domains_edit.php]: Good idea -> done.

**Comment by Guite** [comment]
If the field is hidden for customers, what is the value of this feature at all? Admins can also use their domain management provider to lookup the authcodes.

**Comment by d00p** [comment]
@Guite who says it's hidden for customers? Also it's not even merged into our repository...

**Comment by Guite** [comment]
@d00p okay, misunderstood the conversation above. The field is only hidden if there is no authcode available.

**Comment by chaos234** [comment]
Would like to have this too in the next version or is there smth. wrong with it?

**Comment by d00p** [comment]
Cant remember. I'll Check later.

**Comment by d00p** [comment]
I'm not sure whether this makes sense as an Authcode is often only temporarily valid and there is no interface to any domain-registrar-service

**Comment by chaos234** [comment]
It would make sense because tehn you can request the auth-code from your domain provider - e. g. hetzner - and past it in the interface for the customer and then the interface sends an automated message about an available auth code.

**Comment by d00p** [comment]
I dont see any mailing code in this PR...

**Comment by d00p** [comment]
three year old PR, sorry, close as no response

**Pull request closed without merging**